scripts/linters/css_linter.py

```diff
@@ -58,6 +59,36 @@ def all_filepaths(self):
         """Return all filepaths."""
         return self.files_to_lint
 
+    @staticmethod
+    def _get_trimmed_error_messages(lint_messages):
+        """Remove extra bits from stylelint error messages.
+
+        Args:
+            lint_messages: str. Messages returned by the css linter.
+
+        Returns:
+            str. A string with the trimmed error messages.
+        """
+        error_messages = ''
+        # We need to extract messages from the list and split them line by
+        # line so we can loop through them.
+        messages = lint_messages.split('\n')
```

Comment:
You cannot say that a "lint messages" consists of multiple "message"s.

Also I am a bit confused by the naming. Is every line actually a valid message? If not, then what you have here is wrong and you should use something like "lines" or "css_output_lines" instead.